.circleci/config.yml

```diff
@@ -163,7 +163,8 @@ jobs:
           cd ${HOME}/project/
 
           export DOCKER_IMAGE=soumith/conda-cuda
-          export VARS_TO_PASS="-e PYTHON_VERSION -e BUILD_VERSION -e PYTORCH_VERSION -e UNICODE_ABI -e CU_VERSION"
+          export PYTORCH_TEST_WITH_SLOW=1
```

Comment:
Those changes should be done in `config.yml.in`, and then we should call `python regenerate.py` that will create the `config.yml` for you.

Also, there is a single CI test that runs with this configuration (gpu.medium), which doesn't get tested on Python2 for example.

I think that it might be better to separate this change from this PR, because it might require changing how we handle CI builds.

Reply:
Most of the tests have a pretty significant bump in runtime

Top 10 with `PYTORCH_TEST_WITH_SLOW=0`:
```
63.04s call     test/test_models.py::ModelTester::test_memory_efficient_densenet   
60.13s call     test/test_models.py::ModelTester::test_fasterrcnn_double           
48.19s call     test/test_models.py::ModelTester::test_keypointrcnn_resnet50_fpn   
41.29s call     test/test_models.py::ModelTester::test_maskrcnn_resnet50_fpn       
36.69s call     test/test_models.py::ModelTester::test_fasterrcnn_resnet50_fpn
24.86s call     test/test_models.py::ModelTester::test_vgg19_bn
24.33s call     test/test_models.py::ModelTester::test_vgg16_bn
23.98s call     test/test_models.py::ModelTester::test_vgg13
23.49s call     test/test_models.py::ModelTester::test_vgg11_bn
23.13s call     test/test_models.py::ModelTester::test_deeplabv3_resnet101
```

Top 10 after `PYTORCH_TEST_WITH_SLOW=1`:
```
575.13s call     test/test_models.py::ModelTester::test_densenet121                
276.74s call     test/test_models.py::ModelTester::test_fcn_resnet101              
218.32s call     test/test_models.py::ModelTester::test_inception_v3               
182.89s call     test/test_models.py::ModelTester::test_shufflenet_v2_x1_0         
179.08s call     test/test_models.py::ModelTester::test_mobilenet_v2
171.40s call     test/test_models.py::ModelTester::test_deeplabv3_resnet101
127.75s call     test/test_models.py::ModelTester::test_googlenet
93.93s call     test/test_models.py::ModelTester::test_resnext50_32x4d
70.70s call     test/test_models.py::ModelTester::test_memory_efficient_densenet
```

PyTorch does have a CI job specifically for slow tests, maybe something like that would be better here too. I can separate the CI changes to a different PR, leaving this one to rely on manually setting `PYTORCH_TEST_WITH_SLOW=1` to run the TorchScript tests.

Reply:
Yeah, I agree. Let me add a specific CI test for testing this in a single configuration for now

Reply:
@driazati tests are failing for faster rcnn with precision issues. Can you have a look (FYI I force-pushed to your branch to rebase). I'm looking into configuring CI to run the slow tests